cfme/tests/infrastructure/test_system_image_type.py

```diff
@@ -9,41 +9,49 @@
 pytestmark = [pytest.mark.tier(3)]
 
 
-def test_system_image_type_crud():
+@pytest.fixture(scope="module")
+def collection_init(appliance):
+    return appliance.collections.system_image_type
+
+
+def test_system_image_type_crud(collection_init):
     """
     Tests a System Image Type using CRUD operations.
     """
-    sys_image_type = SystemImageType(
+    collection = collection_init
```

Comment:
**REQUIRED**
Please either rename `collection_init` fixture or delete this redundant line.